e2e/features/adjourn-without-a-date.feature

```diff
@@ -60,6 +60,114 @@ Feature: Adjourn hearing without a date
     And within the `Hearing documents` collection's second item, I should see `-Gonzlez-hearing-notice.PDF` in the `Document` field
 
 
-    
+  @RIA-2960
+  Scenario: Case Officer relists the case after adjourned
 
-    
\ No newline at end of file
+    Given I am signed in as a `Legal Rep`
+    And I create a new case
+    And I save my initial appeal
+    And I submit my appeal
+    And I switch to be a `Case Officer`
+    And I request respondent evidence
+    And I upload respondent evidence
+    And I switch to be a `Legal Rep`
+    And I build my case
+    And I submit my case
+    And I switch to be a `Case Officer`
+    And I request respondent review
+    And I add the appeal response
+    And I request appellant review
+    And I request hearing requirements
+    And I switch to be a `Legal Rep`
+    And I submit hearing requirements with all yes
+    And I switch to be a `Case Officer`
+    And I record agreed hearing requirements yes path
+    And I switch to be a `Admin Officer`
+    And I list the case
```

Comment:
This lines can be removed because they are used in the Background section, right?

Reply:
Yes, this has been resolved :)